src/libs/PolicyUtils.ts

```diff
@@ -85,6 +86,13 @@ function hasPolicyCategoriesError(policyCategories: OnyxEntry<PolicyCategories>)
     return Object.keys(policyCategories ?? {}).some((categoryName) => Object.keys(policyCategories?.[categoryName]?.errors ?? {}).length > 0);
 }
 
+/**
+ * Checks if the policy has auto-sync error.
+ */
+function hasAutoSyncError(policy: OnyxEntry<Policy>): boolean {
```

Comment:
```suggestion
function hasSyncError(policy: OnyxEntry<Policy>): boolean {
```

Let's remove "auto" references since this affects both auto and manual synchronization.